includes/admin/plugins.php

```diff
@@ -142,11 +142,8 @@ function perflab_render_plugin_card( array $plugin_data ) {
 	$version     = $plugin_data['version'];
 	$name        = wp_strip_all_tags( $title . ' ' . $version );
 
-	$requires_php = isset( $plugin_data['requires_php'] ) ? $plugin_data['requires_php'] : null;
-	$requires_wp  = isset( $plugin_data['requires'] ) ? $plugin_data['requires'] : null;
-
-	$compatible_php = is_php_version_compatible( $requires_php );
-	$compatible_wp  = is_wp_version_compatible( $requires_wp );
+	$compatible_php = is_php_version_compatible( $plugin_data['requires_php'] );
+	$compatible_wp  = is_wp_version_compatible( $plugin_data['requires'] );
```

Comment:
@mrmakadia94 Actually, that last [suggestion](https://github.com/WordPress/performance/pull/1168#discussion_r1576963271), while it works, it technically is not correct because `is_php_version_compatible()` and `is_wp_version_compatible()` are only supposed to take strings. So I believe this is right:

```suggestion
	$compatible_php = ! $plugin_data['requires_php'] || is_php_version_compatible( $plugin_data['requires_php'] );
	$compatible_wp  = ! $plugin_data['requires'] || is_wp_version_compatible( $plugin_data['requires'] );
```

Reply:
@westonruter please check now, i updated the PR.